### PR TITLE
[ES] Native review of `History of osu!/2015`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/2015/es.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+Se agregó soporte para el [Wii Remote](https://es.wikipedia.org/wiki/Wii_Remote) para [osu!taiko](/wiki/Game_mode/osu!taiko) el 26 de marzo de 2015.[^osu-weekly-3]
 
-El 31 de marzo de 2015, se agregaron las [osu!coins](/wiki/History_of_osu!/April_Fools#osu!coins) como moneda de broma en el juego para osu! como parte de una broma del día de los Inocentes. Las monedas se crearon para parodiar a los juegos móviles [free-to-play](https://en.wikipedia.org/wiki/Free-to-play) y su uso común de monedas en el juego y [microtransacciones](https://en.wikipedia.org/wiki/Microtransaction). Las osu!coins se retiraron rápidamente al día siguiente.
+El 31 de marzo de 2015, se agregaron las [osu!coins](/wiki/History_of_osu!/April_Fools#osu!coins) como moneda de broma en el juego para osu! como parte de una broma del día de los Inocentes. Las monedas se crearon para parodiar a los juegos móviles [free-to-play](https://es.wikipedia.org/wiki/Videojuego_gratuito) y su uso común de monedas en el juego y [microtransacciones](https://en.wikipedia.org/wiki/Microtransaction). Las osu!coins se retiraron rápidamente al día siguiente.
```
Comment (JuanM4T) [inline wiki/History_of_osu!/2015/es.md:24]:
```suggestion
El 31 de marzo de 2015, se agregaron las [osu!coins](/wiki/History_of_osu!/April_Fools#osu!coins) como moneda de broma en el juego para osu! como parte de una broma del día de los Inocentes. Las monedas se crearon para parodiar a los juegos móviles [free-to-play](https://es.wikipedia.org/wiki/Videojuego_gratuito) y su uso común de monedas en el juego y [microtransacciones](https://es.wikipedia.org/wiki/Microtransacción). Las osu!coins se retiraron rápidamente al día siguiente.
```

**Review by JuanM4T — CHANGES_REQUESTED** [review]

**Review by JuanM4T — APPROVED** [review]

File: wiki/History_of_osu!/2015/es.md [hunk 2]
```diff
@@ -36,13 +37,13 @@ El [mod Co-op](/wiki/Gameplay/Game_modifier/Co-op) se convirtió en un mod no cl
...
 
-El [canal de YouTube de osu!news](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ) subió su [primer vídeo](https://www.youtube.com/watch?v=KQbudVxEjr8) al público el 8 de mayo de 2015. El canal estaba a cargo de varios miembros de la comunidad y tenía como objetivo crear vídeos sobre exhibiciones de skins, torneos, mapeo, concursos y otras noticias de la comunidad.[^osu-weekly-9][^osu-news-welcome]
+El [canal de YouTube de osu!news](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ) subió su [primer vídeo](https://www.youtube.com/watch?v=KQbudVxEjr8) al público el 8 de mayo de 2015. El canal estaba a cargo de varios miembros de la comunidad y tenía como objetivo crear vídeos sobre exhibiciones de skins, torneos, mapping, concursos y otras noticias de la comunidad.[^osu-weekly-9][^osu-news-welcome]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:42]:
```suggestion
El [canal de YouTube osu!news](https://www.youtube.com/channel/UCZKQIqv9O2tddMNUMAxWaqQ) subió su [primer vídeo](https://www.youtube.com/watch?v=KQbudVxEjr8) al público el 8 de mayo de 2015. El canal estaba a cargo de varios miembros de la comunidad y tenía como objetivo crear vídeos sobre exhibiciones de skins, torneos, mapping, concursos y otras noticias de la comunidad.[^osu-weekly-9][^osu-news-welcome]
```

File: wiki/History_of_osu!/2015/es.md [hunk 3]
```diff
@@ -36,13 +37,13 @@ El [mod Co-op](/wiki/Gameplay/Game_modifier/Co-op) se convirtió en un mod no cl
...
 
-El teclado osu! (también conocido como el "teclado osu! 'nono'") se agregó a la osu!store el 30 de mayo de 2015, con un precio de venta minorista de alrededor de $40 en el lanzamiento. Anunciado inicialmente en un [tweet de peppy](https://twitter.com/ppy/status/603797988742336512) el 28 de mayo de 2015, solo había 100 teclados disponibles en el lanzamiento y se parecían más a un keypad que a un teclado de tamaño completo.[^osu-weekly-12][^ppy-tweet-osu-keyboards]
+El teclado osu! (también conocido como el «teclado osu! 'nono'») se agregó a la osu!store el 30 de mayo de 2015, con un precio de venta minorista de alrededor de $40 en el lanzamiento. Anunciado inicialmente en un [tuit de peppy](https://twitter.com/ppy/status/603797988742336512) el 28 de mayo de 2015, solo había 100 teclados disponibles en el lanzamiento y se parecían más a un keypad que a un teclado de tamaño completo.[^osu-weekly-12][^ppy-tweet-osu-keyboards]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:44]:
```suggestion
El teclado osu! (también conocido como el «teclado osu! 'nono'») se agregó a la osu!store el 30 de mayo de 2015, con un precio de venta minorista de alrededor de 40 $ en el lanzamiento. Anunciado inicialmente en un [tuit de peppy](https://twitter.com/ppy/status/603797988742336512) el 28 de mayo de 2015, solo había 100 teclados disponibles en el lanzamiento y se parecían más a un keypad que a un teclado de tamaño completo.[^osu-weekly-12][^ppy-tweet-osu-keyboards]
```

File: wiki/History_of_osu!/2015/es.md
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:1]: en la línea 50 pon `4 $`/`100 $`

Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:1]: 56 osu team

File: wiki/History_of_osu!/2015/es.md [hunk 4]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
 
 El 15 de julio de 2015, se anunció que se creó un nuevo equipo público de [Slack](https://slack.com/) con el fin de mejorar la comunicación con los contribuyentes públicos. [peppy](https://osu.ppy.sh/users/2) inicialmente hizo el anuncio en su blog y explicó que cualquiera que estuviera dispuesto a ayudar a probar funciones de cutting-edge, contribuir con recursos gráficos/de sonido u obtener ayuda trabajando con osu! podrían venir y unirse al equipo y hablar con la gente de diferentes [equipos de osu!](/wiki/People/osu!_team) directamente.[^ppy-blog-july-15]
 
-*Nota: Todavía había un equipo privado de Slack que el equipo de osu! utilizó para comunicaciones internas, que no era accesible o visible para usuarios externos; estos dos equipos eran independientes entre sí.*
+*Nota: También existía un equipo privado de Slack que el osu! team utilizó para comunicaciones internas, que no era accesible o visible para usuarios externos; estos dos equipos eran independientes entre sí.*
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:58]:
```suggestion
*Nota: También existía un equipo privado de Slack que el osu! team usó para comunicaciones internas, que no era accesible o visible para usuarios externos; estos dos equipos eran independientes entre sí.*
```

File: wiki/History_of_osu!/2015/es.md [hunk 5]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
 
-No obstante, el nuevo equipo era completamente público y estaba disponible para que cualquiera se uniera, lo que provocó que muchos usuarios no deseados inundaran los canales. Esto obligó a peppy a "bloquear" al nuevo equipo al día siguiente. A pesar de esto, peppy siguió haciendo uso del equipo durante el resto del año y, a menudo, invitaba a los usuarios a venir y unirse cuando estaban probando nuevas funciones o implementaciones.[^ppy-blog-july-16][^ppy-blog-july-20]
+No obstante, el nuevo equipo era completamente público y estaba disponible para que cualquiera se uniera, lo que provocó que muchos usuarios no deseados inundaran los canales. Esto obligó a peppy a «bloquear» el nuevo equipo al día siguiente. A pesar de esto, peppy siguió haciendo uso del equipo durante el resto del año y, a menudo, invitaba a los usuarios a venir y unirse cuando estaban probando nuevas funciones o implementaciones.[^ppy-blog-july-16][^ppy-blog-july-20]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:60]:
```suggestion
No obstante, el nuevo equipo era completamente público y estaba disponible para que cualquiera se uniera, lo que provocó que muchos usuarios no deseados inundaran los canales. Esto obligó a peppy a «bloquear» el nuevo equipo hasta el día siguiente. A pesar de esto, peppy siguió haciendo uso del equipo durante el resto del año y, a menudo, invitaba a los usuarios a venir y unirse cuando estaban probando nuevas funciones o implementaciones.[^ppy-blog-july-16][^ppy-blog-july-20]
```

File: wiki/History_of_osu!/2015/es.md [hunk 6]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
+El antiguo contador de FPS fue reemplazado por una nueva «pantalla de actualización y latencia» el 2 de septiembre de 2015. Esta nueva pantalla aún mostraría los FPS actuales (fotogramas por segundo), pero el nuevo cambio hizo que se ocultara de la vista si superaba los 500 FPS. Además, se agregó un nuevo contador debajo del contador de FPS que mostraría exactamente cuánta latencia se estaba introduciendo durante la representación de cuadros (medida en milisegundos). Los fotogramas «tartamudeados» o «caídos» se mostrarían como rectángulos de colores a la izquierda del contador para visualizar mejor exactamente cuánta latencia de procesamiento de fotogramas estaba experimentando un usuario.[^ppy-blog-sep-mwr][^osu-weekly-29]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:68]:
```suggestion
El antiguo contador de FPS fue reemplazado por una nueva «pantalla de actualización y latencia» el 2 de septiembre de 2015. Esta nueva pantalla aún mostraría los FPS (fotogramas por segundo) actuales, pero el nuevo cambio hizo que se ocultara de la vista si superaba los 500 FPS. Además, se agregó un nuevo contador debajo del contador de FPS que mostraría exactamente cuánta latencia se estaba introduciendo durante la representación de cuadros (medida en milisegundos). Los fotogramas «tartamudeados» o «caídos» se mostrarían como rectángulos de colores a la izquierda del contador para visualizar mejor exactamente cuánta latencia de procesamiento de fotogramas estaba experimentando un usuario.[^ppy-blog-sep-mwr][^osu-weekly-29]
```

File: wiki/History_of_osu!/2015/es.md [hunk 7]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
+El 26 de septiembre de 2015, [cYsmix](http://cysmix.com/) se convirtió en uno de los primeros artistas oficiales de osu!. Esta asociación implicó la creación de tres pistas hechas específicamente para osu!, junto con la creación de un concurso de mapping usando esas pistas. Las tres pistas antes mencionadas fueron las siguientes:
 
-- ["Manic"](https://soundcloud.com/olemlanglie/cysmix-manic-osu)
-- ["House with Legs"](https://soundcloud.com/olemlanglie/cysmix-house-with-legs-osu)
-- ["Dovregubben's Hall (New Ver.)"](https://soundcloud.com/olemlanglie/cysmix-dovregubbens-hall-new-ver-osu)
+- [«Manic»](https://soundcloud.com/olemlanglie/cysmix-manic-osu)
+- [«House with Legs»](https://soundcloud.com/olemlanglie/cysmix-house-with-legs-osu)
+- [«Dovregubben's Hall (New Ver.)»](https://soundcloud.com/olemlanglie/cysmix-dovregubbens-hall-new-ver-osu)
 
 ![](img/cysmix-album-cover-1.jpg "La portada del álbum para el primer lanzamiento de canciones oficiales de osu! de cYsmix")
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:76]:
```suggestion
![](img/cysmix-album-cover-1.jpg "La portada del álbum para el primer lanzamiento de canciones oficiales de osu! por cYsmix")
```

File: wiki/History_of_osu!/2015/es.md [hunk 8]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
+Además de esto, se creó una «recompensa abierta» llamada «Mapping With Rewards» para este evento, que permitía a cualquiera mapear una de las pistas antes mencionadas, ponerla en un estado clasificable y, si era lo suficientemente impresionante, esa persona podría recibir una recompensa por ello. La recompensa ofrecida en ese momento era la siguiente: una canción incluida con el cliente (por un período de tiempo limitado), crédito de $50 en la osu!store, 6 meses de supporter y una insignia de perfil única. Según los informes, esta recompensa abierta se creó debido a la necesidad de recompensar a los jugadores por beatmaps de calidad y para celebrar las nuevas pistas sin tener que pasar por la molestia que conlleva la ejecución de concursos de mapping tradicionales.[^ppy-blog-sep-mwr]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:80]:
```suggestion
Además de esto, se creó una «recompensa abierta» llamada «Mapping With Rewards» para este evento, que permitía a cualquiera mapear una de las pistas antes mencionadas, ponerla en un estado clasificable y, si era lo suficientemente impresionante, esa persona podría recibir una recompensa por ello. La recompensa ofrecida en ese momento era la siguiente: una canción incluida con el cliente (por un período de tiempo limitado), crédito de 50 $ en la osu!store, 6 meses de supporter y una insignia de perfil única. Según los informes, esta recompensa abierta se creó debido a la necesidad de recompensar a los jugadores por beatmaps de calidad y para celebrar las nuevas pistas sin tener que pasar por la molestia que conlleva la ejecución de concursos de mapping tradicionales.[^ppy-blog-sep-mwr]
```

File: wiki/History_of_osu!/2015/es.md [hunk 9]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
 
-Sin embargo, a pesar de [las promesas de peppy en su blog](https://blog.ppy.sh/post/132009865043/mapping-with-rewards-oct-2015), los resultados del Mapeo con recompensas no se publicaron al final de el año.
+Sin embargo, a pesar de [las promesas de peppy en su blog](https://blog.ppy.sh/post/132009865043/mapping-with-rewards-oct-2015), los resultados del Mapping With Rewards no se publicaron al final del año.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:82]:
```suggestion
Sin embargo, a pesar de [las promesas de peppy en su blog](https://blog.ppy.sh/post/132009865043/mapping-with-rewards-oct-2015), los resultados de Mapping With Rewards no se publicaron al final del año.
```

File: wiki/History_of_osu!/2015/es.md [hunk 10]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
+Después de meses de trabajo, se completó la migración de [DirectX](https://es.wikipedia.org/wiki/DirectX) a [OpenGL](https://es.wikipedia.org/wiki/OpenGL) y se envió a todos los usuarios en el flujo de lanzamiento Stable el 16 de octubre de 2015. Sin embargo, los usuarios que no pudieron ejecutar la nueva actualización debido a problemas de compatibilidad cambiaron a una nueva secuencia de lanzamiento llamada «Stable (Fallback)», que usaba el antiguo renderizado de gráficos de DirectX. Estaba destinado a ser compatible continuamente junto con el lanzamiento de Stable hasta que el osu! team confiara lo suficiente en la compatibilidad del renderizado OpenGL para todos los usuarios.[^ppy-blog-june-11][^ppy-blog-july-31][^ppy-blog-august-12][^ppy-blog-september-04][^ppy-blog-october-16]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:86]:
```suggestion
Después de meses de trabajo, se completó la migración de [DirectX](https://es.wikipedia.org/wiki/DirectX) a [OpenGL](https://es.wikipedia.org/wiki/OpenGL) y se envió a todos los usuarios en el flujo de lanzamiento Stable el 16 de octubre de 2015. Sin embargo, los usuarios que no pudieron ejecutar la nueva actualización debido a problemas de compatibilidad cambiaron a un nuevo canal de lanzamiento llamado «Stable (Fallback)», que usaba el antiguo renderizado de gráficos de DirectX. Estaba destinado a ser compatible continuamente junto con el lanzamiento de Stable hasta que el osu! team confiara lo suficiente en la compatibilidad del renderizado OpenGL para todos los usuarios.[^ppy-blog-june-11][^ppy-blog-july-31][^ppy-blog-august-12][^ppy-blog-september-04][^ppy-blog-october-16]
```

File: wiki/History_of_osu!/2015/es.md [hunk 11]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
 
-Otro lote de canciones oficiales de osu! fueron lanzadas por cYsmix el 27 de octubre de 2015, y también se abrió un concurso similar de "estilo de recompensa abierta". Las tres canciones fueron ["Fright March"](https://soundcloud.com/olemlanglie/cysmix-fright-march-osu), ["Moonlight Sonata"](https://soundcloud.com/olemlanglie/cysmix-moonlight-sonata-osu), y ["Classic Pursuit"](https://soundcloud.com/olemlanglie/cysmix-classic-pursuit-osu).[^ppy-blog-oct-mwr][^osu-weekly-34]
+Otro lote de canciones oficiales de osu! fueron lanzadas por cYsmix el 27 de octubre de 2015, y también se abrió un concurso similar de «estilo de recompensa abierta». Las tres canciones fueron [«Fright March»](https://soundcloud.com/olemlanglie/cysmix-fright-march-osu), [«Moonlight Sonata»](https://soundcloud.com/olemlanglie/cysmix-moonlight-sonata-osu), y [«Classic Pursuit»](https://soundcloud.com/olemlanglie/cysmix-classic-pursuit-osu).[^ppy-blog-oct-mwr][^osu-weekly-34]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:88]:
```suggestion
Otro lote de canciones oficiales de osu! fueron lanzadas por cYsmix el 27 de octubre de 2015, y también se abrió un concurso similar al del «estilo de recompensa abierta». Las tres canciones fueron [«Fright March»](https://soundcloud.com/olemlanglie/cysmix-fright-march-osu), [«Moonlight Sonata»](https://soundcloud.com/olemlanglie/cysmix-moonlight-sonata-osu) y [«Classic Pursuit»](https://soundcloud.com/olemlanglie/cysmix-classic-pursuit-osu).[^ppy-blog-oct-mwr][^osu-weekly-34]
```

File: wiki/History_of_osu!/2015/es.md [hunk 12]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
 
 ![](img/cysmix-album-cover-2.jpg "La portada del álbum para el segundo lanzamiento de canciones oficiales de osu! de cYsmix")
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:90]:
```suggestion
![](img/cysmix-album-cover-2.jpg "La portada del álbum para el segundo lanzamiento de canciones oficiales de osu! por cYsmix")
```

File: wiki/History_of_osu!/2015/es.md [hunk 13]
```diff
@@ -54,41 +55,41 @@ La capacidad de ver cómo se vería una repetición sin [mods](/wiki/Gameplay/Ga
...
 
-Al igual que el [concurso de Mapeo con recompensas del mes anterior](#septiembre), los resultados no se publicaron antes de fin de año.
+Al igual que el [concurso de Mapping With Rewards del mes anterior](#septiembre), los resultados no se publicaron antes de fin de año.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:92]:
```suggestion
Al igual que el [concurso Mapping With Rewards del mes anterior](#septiembre), los resultados no se publicaron antes de fin de año.
```

File: wiki/History_of_osu!/2015/es.md [hunk 14]
```diff
@@ -102,10 +103,10 @@ Los banners de perfil de la [OWC](/wiki/Tournaments#osu!-world-cup) se agregaron
...
 
-> Me ha llamado la atención que en los últimos días una gran cantidad de streamers de osu! han sido hackeados. El vínculo común parece ser que todos tenían cuentas xsplit activas (que fue hackeado en 2013, filtrando todas las contraseñas). Esto significa que dichos usuarios estaban usando la misma contraseña en twitch, twitter, osu! y más. En primer lugar, si está haciendo algo como esto, SALGA AHORA MISMO Y CAMBIE SUS CONTRASEÑAS. ¡Obtenga un administrador de contraseñas y configure una contraseña por servicio!
+> Me ha llamado la atención que en los últimos días una gran cantidad de streamers de osu! han sido hackeados. El vínculo común parece ser que todos tenían cuentas xsplit activas (que fue hackeado en 2013, filtrando todas las contraseñas). Esto significa que dichos usuarios estaban usando la misma contraseña en twitch, twitter, osu! y más. En primer lugar, si estás haciendo algo como esto, SAL AHORA MISMO Y CAMBIA TUS CONTRASEÑAS. ¡Obtén un administrador de contraseñas y configura una contraseña por servicio!
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:106]:
```suggestion
> Me ha llamado la atención que en los últimos días una gran cantidad de streamers de osu! han sido hackeados. El vínculo común parece ser que todos tenían cuentas de xsplit activas (que fue hackeado en 2013, filtrando todas las contraseñas). Esto significa que dichos usuarios estaban usando la misma contraseña en twitch, twitter, osu! y más. En primer lugar, si estás haciendo algo como esto, SAL AHORA MISMO Y CAMBIA TUS CONTRASEÑAS. ¡Obtén un administrador de contraseñas y configura una contraseña por servicio!
```

File: wiki/History_of_osu!/2015/es.md [hunk 15]
```diff
@@ -102,10 +103,10 @@ Los banners de perfil de la [OWC](/wiki/Tournaments#osu!-world-cup) se agregaron
...
 > Aunque estoy seguro de que osu! no estuvo involucrado en la filtración de ningún detalle, me aseguro de que más intentos de este tipo no resulten en cuentas de osu! comprometidas. A partir de ayer comencé a implementar la verificación de correo electrónico para ciertas acciones de la cuenta. En el futuro, también agregaremos verificación de correo electrónico al cliente del juego, pero esto requerirá un poco más de implementación. Espéralo en un futuro cercano.
 
-— peppy, "20151117"[^osu-weekly-36]
+— peppy, «20151117»[^osu-weekly-36]
 
 Unos días después (19 de noviembre de 2015), peppy actualizó a los usuarios sobre la situación en [otra publicación de blog](https://blog.ppy.sh/post/133524244723/20151119), en la que explicó la situación y mantuvo su postura igual:
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:111]:
```suggestion
Unos días después (19 de noviembre de 2015), peppy actualizó a los usuarios sobre la situación en [otra publicación en su blog](https://blog.ppy.sh/post/133524244723/20151119), en la que explicó la situación y mantuvo su postura igual:
```

File: wiki/History_of_osu!/2015/es.md [hunk 16]
```diff
@@ -1,26 +1,27 @@
...
+Las primeras pegatinas de los planos de beatmaps de osu! se agregaron a la [osu!store](https://osu.ppy.sh/store/listing) el 20 de marzo de 2015. Las pegatinas venían en dos paquetes de hojas de pegatinas personalizadas de 8,5 x 11 con elementos como círculos, contadores de combo, sliders, y mucho más.[^osu-stickers] Para celebrar el lanzamiento, se llevó a cabo un concurso para ver quién podía hacer el mejor [beatmap](/wiki/Beatmap) usando las pegatinas, con un premio de primer lugar de una osu! tablet y un juego de pegatinas osu! 1 y 2. A pesar de esto, el concurso se canceló debido a la falta de presentaciones, y en ese momento nunca se declaró públicamente ningún seguimiento al respecto.[^osu-stickers][^osu-stickers-submission-count]
 
-Se agregó soporte para el [Wii Remote](https://es.wikipedia.org/wiki/Wii_Remote) para [osu!taiko](/wiki/Game_mode/osu!taiko) el 26 de marzo de 2015.[^osu-weekly-3] <!--changelog entry missing-->
+Se agregó soporte para el [Wii Remote](https://es.wikipedia.org/wiki/Wii_Remote) para [osu!taiko](/wiki/Game_mode/osu!taiko) el 26 de marzo de 2015.[^osu-weekly-3]
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:22]:
```suggestion
Se agregó soporte para el [Wii Remote](https://es.wikipedia.org/wiki/Wii_Remote) en [osu!taiko](/wiki/Game_mode/osu!taiko) el 26 de marzo de 2015.[^osu-weekly-3]
```

File: wiki/History_of_osu!/2015/es.md [hunk 17]
```diff
@@ -1,26 +1,27 @@
 ---
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:1]: en la línea 30 pon "no clasificable"

File: wiki/History_of_osu!/2015/es.md [hunk 18]
```diff
@@ -1,32 +1,33 @@
...
```
Comment (agatemosu) [inline wiki/History_of_osu!/2015/es.md:1]: en el 36 pon ...que anunciaba el nuevo sistema de...

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**